tests/integration/conftest.py

```diff
@@ -7,37 +10,42 @@
 from adcm_aio_client.core.client import ADCMClient, build_client
 from adcm_aio_client.core.types import Credentials
 from tests.integration.setup_environment import (
+    DB_USER,
     ADCMContainer,
     ADCMPostgresContainer,
+    DatabaseInfo,
     adcm_image_name,
-    db_name,
-    db_password,
-    db_user,
     postgres_image_name,
 )
 
+BUNDLES = Path(__file__).parent / "bundles"
+
 
 @pytest.fixture(scope="session")
 def network() -> Generator[Network, None, None]:
     with Network() as network:
         yield network
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="session")
```

Comment:
wouldn't it mean to isolation issues between tests?

Reply:
no, each test will have it's own database within 1 postgres instance (for now)